packages/core/src/dialects/cockroachdb/data-types.ts

```diff
@@ -0,0 +1,7 @@
+import * as DataTypes from '../postgres/data-types';
+
+DataTypes.GEOGRAPHY.prototype.getBindParamSql = (value, options): string => {
```

Comment:
You can't modify another dialect like this

You should either:

- make your changes in the postgres data type itself if it's compatible
- or extend that DataType and replace the method

Reply:
Gotcha thanks!